ceph/deb.sh

```diff
@@ -3,12 +3,16 @@ set -eu
 
 CEPH_DIR=$(readlink -f $(dirname $0))
 
-if [ $# -ne 1 ]; then
-    echo "deb.sh VERSION"
+if [ $# -lt 1 ] || [ $# -gt 2 ]; then
```

Comment:
I think it is better to use `getopts` for optional arguments.

Reply:
Is it OK to use `getopts` for both `VERSION` and `WITH_ASAN` parameters?
I think that mixing different way of getting parameters (I mean, one is using `getopts' and the other is not) results in the complex parameter parsing logic.
For example, the following command works well.
```
./deb.sh -a 16.2.7
```
On the other hand, the following one doesn't.
```
./deb.sh 16.2.7 -a 
```

So I think the following one is good.
```
./deb.sh -v 16.2.7 -a    # OK!
./deb.sh -a -v 16.2.7    # OK!
```

Could you give me a comment about it?

Reply:
The version parameter is mandatory, so I prefer `deb.sh -a 16.2.7` to `deb.sh -a -v 16.2.7`.
Many cli programs use `prog [opt...] arg...` style, I think it is unnecessary to support `deb.sh 16.2.7 -a`.

Reply:
Thank you for your comment. I will adopt `prog [opt...] arg...` style.